.github/workflows/release.yml

```diff
@@ -1,12 +1,14 @@
 name: Release
 
 on:
+  schedule:
+    # Every Friday at 19:00 (UTC)
+    - cron: '00 19 * * FRI'
```

Comment:
I'm confused with this PR - is this releasing every week via this cron schedule? I'm sure that isn't what its doing, but without comments in here explaining, it is confusing.

Reply:
This PR adds the schedulling mechanism to the Helm charts release pipeline. I'm not sure when you say that it isn't what is doing. 

This PR adds the cron, the release type selection (checking if it needs to skip the release if we are not in the end sprint week) and also adds some checks similar to the server and operator ones.

Reply:
I added the "do not merge" label to wait after the next release that will be manual.

Reply:
I thought we did this every 3 weeks. This says in the comment it does this every Friday. So I guess it determines later if it is in the third week to actually do the work (the other 2 weeks it just does nothing?)

Reply:
Yes, with a cron sentence you can't fire every 3 weeks, it was commented on the following PR too: https://github.com/kiali/kiali/pull/5037 (comment #2).

So, I don't like the fact that we will see a release pipeline run for nothing, but it's the way GH Actions works today.